server/@types/ui/index.d.ts

```diff
@@ -374,6 +374,7 @@ export type UserDetails = {
   permissions: Array<ApprovedPremisesUserPermission>
   active: boolean
   apArea: ApArea
+  version: number
```

Comment:
should this be a `string`?

Reply:
i think number is fine, we convert an integer to a string when writing this to the header in the API

Reply:
If the UI is only dealing with the header value it would be better to have this typed as string, to avoid excessive type conversions.